src/site/content/en/blog/responsive-web-design-basics/index.md

```diff
@@ -0,0 +1,579 @@
+* [Use CSS media queries for responsiveness](#media-queries)
+* [How to choose breakpoints](#breakpoints)
+* [View media query breakpoints in Chrome DevTools](#devtools)
+
+The use of mobile devices to surf the web continues to grow at an astronomical pace, and these devices are often constrained by display size and require a different approach to how content is laid out on the screen.
+
+Responsive web design, originally defined by [Ethan Marcotte in A List Apart](http://alistapart.com/article/responsive-web-design/), responds to the needs of the users and the devices they're using. The layout changes based on the size and capabilities of the device. For example, on a phone users would see content shown in a single column view; a tablet might show the same content in two columns.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="resize.webm" type="video/webm; codecs=vp8">
+    <source src="resize.mp4" type="video/mp4; codecs=h264">
+  </video>
+</figure>
+
+A multitude of different screen sizes exist across phones, "phablets," tablets, desktops, game consoles, TVs, and even wearables. Screen sizes are always changing, so it's important that your site can adapt to any screen size, today or in the future. In addition, devices have different features with which we interact with them. For example some of your visitors will be using a touchscreen. Modern responsive design considers all of these things to optimize the experience for everyone.
+
+
+## Set the viewport {: #viewport }
+
+Pages optimized for a variety of devices must include a meta viewport tag in the head of the document.
+A meta viewport tag gives the browser instructions on how to control the page's dimensions and scaling.
+
+To attempt to provide the best experience, mobile browsers render
+the page at a desktop screen width (usually about `980px`, though this varies
+across devices), and then try to make the content look better by increasing
+font sizes and scaling the content to fit the screen.
+This means that font sizes may appear inconsistent to users,
+who may have to double-tap or pinch-to-zoom
+in order to see and interact with the content.
+
+```html/4
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    …
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    …
+  </head>
+  …
+
+```
+
+Using the meta viewport value `width=device-width` instructs the page to match
+the screen's width in device-independent pixels. A device (or density) independent pixel being a representation of a single pixel, which may on a high density screen consist of many physical pixels. This allows the page to reflow
+content to match different screen sizes, whether rendered on a small mobile
+phone or a large desktop monitor.
+
+<figure class="w-figure">
+  <img src="./without-vp-meta.png" width="440" alt="Screenshot of a page with the text hard to read as it is very zoomed out">
```

Comment:
This looks grainy on my MacBook Air. Can you re-upload? `1600px` width is safe for ensuring that it has high resolution on all screens (because our `max-width` is `800px`).

![image](https://user-images.githubusercontent.com/4713486/82072900-42977a00-968d-11ea-90e9-920e3cdd891f.png)

Reply:
these are small images as they are from phone sizes within DevTools. I've redone it however.